tests/release/conftest.py

```diff
@@ -0,0 +1,158 @@
+import functools
+import os
+import time
+from collections.abc import Callable
+from pathlib import Path
+from typing import Any
+
+import pandas as pd
+import pytest
+from memory_profiler import memory_usage  # type: ignore
+
+from pseudopeople.interface import (
+    generate_american_community_survey,
+    generate_current_population_survey,
+    generate_decennial_census,
+    generate_social_security,
+    generate_taxes_1040,
+    generate_taxes_w2_and_1099,
+    generate_women_infants_and_children,
+)
+
+DATASET_GENERATION_FUNCS: dict[str, Callable[..., Any]] = {
+    "census": generate_decennial_census,
+    "acs": generate_american_community_survey,
+    "cps": generate_current_population_survey,
+    "ssa": generate_social_security,
+    "tax_w2_1099": generate_taxes_w2_and_1099,
+    "wic": generate_women_infants_and_children,
+    "tax_1040": generate_taxes_1040,
+}
+
+DEFAULT_YEAR = 2020
```

Comment:
nit: Would `CLI_DEFAULT_*` be a more clear name for these constants? That would differentiate between defaults being used directly in the pytests or actual code vs these kinda new-weird command line args.